app/views/trainees/diversity/ethnic_groups/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= content_for(:breadcrumbs) do %>
+  <%= render_component GovukComponent::BackLink.new(
+    text: "Back",
+    href: trainee_path(@trainee)
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= form_with(model: @ethnic_group, url: trainee_diversity_ethnic_group_path(@trainee), method: :put, local: true) do |f| %>
+      <%= f.govuk_error_summary %>
+
+        <%= f.govuk_radio_buttons_fieldset(:ethnic_group, legend: { text: "What is their ethnic group?", tag: "h1", size: "l" }) do %>
+          <% format_ethnic_group_options(Trainee.ethnic_groups.keys).each_with_index do |ethnic_option, index| %>
+            <%= f.govuk_radio_button(
+              :ethnic_group, 
+              ethnic_option,
+              hint: { text: I18n.t("views.forms.ethnic_groups.descriptions.#{ethnic_option}") },
+              label: { text: I18n.t("views.forms.ethnic_groups.labels.#{ethnic_option}"), size: "s" },
+              link_errors: (true if index == 0)) %>
```

Comment:
can we not just put true, just curious, its not like the item in question needs to be link but the whole fieldset

Reply:
Hey, sorry to jump in, but the design system states that the error summary should link to the [first entry in the list](https://design-system.service.gov.uk/components/error-summary/#linking-from-the-error-summary-to-each-answer).

> For questions that require a user to select one or more options from a list using radios or checkboxes, link to the first radio or checkbox.

It's a bit of a pain to handle in the `_fieldset` helpers because they're totally flexible.

That being said, the above _could_ be shortened a bit to `link_errors: index.zero?`

Reply:
done https://github.com/DFE-Digital/register-trainee-teachers/pull/109/commits/a5b78503493c22445fde1ae5f57cb692247d7a22